Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs

```diff
@@ -102,6 +99,10 @@ public void UpdateData()
                 // layouting
                 int j = i % rows;
                 Collider collider = interactive.gameObject.GetComponent<Collider>();
+                if (collider == null)
```

Comment:
Should we also add `Collider` to the list of required components?

Reply:
That would be the Collider in InteractiveToggle. I can attach the InteractiveToggle-script to any GameObject and have the collider in some child of that GameObject, so I don't think having it as required component is a good idea... we can however ignore the collider boundary size for layouting if it is not present. If you don't have a collider attached directly the user can define the distance between the InteractiveToggles by setting the Offsets-values. Alternatively I could iterate over transform.children recursively and calculate the biggest bounding box, but that seems a bit overkill.